mailthon/headers.py

```diff
@@ -74,7 +76,13 @@ def prepare(self, mime):
             if key == 'Bcc' or key == 'Resent-Bcc':
                 continue
             del mime[key]
-            mime[key] = self[key]
+            # python 3.* email's compatibility layer will handle unicode field values in proper way
+            # but python 2.* -- won't (it will encode not only additional field values but also all header value)
+            if int(sys.version[0]) == 2:
```

Comment:
Maybe it's a premature optimization but this should ideally be put in a constant, e.g. ``IS_PY3``. Partially because it would be called in a loop, and global lookups (`sys`) are expensive.